packages/xinit.rb

```diff
@@ -20,12 +20,14 @@ class Xinit < Package
      x86_64: '5df3455157f222c07641d3ff605ff9365fbf4a19d014c2db09999bf84d128296',
   })
 
-  depends_on 'xorg_lib'
+  depends_on 'xclock'
+  depends_on 'xterm'
+  depends_on 'twm'
```

Comment:
Maybe these apps should have xinit as their dependency instead?

Reply:
Nope. Read above.

Reply:
I don't think all these "dependencies" are needed, especially xclock and xterm.  You might be able to make a case for twm, however, the window manager really should be up to the user to decide.  I agree xauth is needed.  Have you tried installing xinit on a fresh Chromebrew yet?

Reply:
`startx` throws errors if those commands aren't found. As `xinit` isn't depended on by any package, it shouldn't matter if it has a few extra dependencies.
Yes, I have installed `xinit` on a fresh Chromebrew installation. It works fine, and does not give errors about missing `xterm`, `xclock`, `xauth`, or `twm`.

Reply:
You are contradicting yourself.  First you say it throws errors and then you say it works without the dependencies?  I'm confused.

Reply:
`startx` gives errors about `<program>: not found` if you don't have `xterm`, `xclock`, `xauth`, and `twm` installed. When I did a fresh Chromebrew installation, I copied those packages and `xinit` over to `/usr/local/lib/crew/packages/`, and installed them. My modified `xinit` package works fine on a fresh Chromebrew installation.

Reply:
I don't think you understand what I'm asking.  Did you try to install xinit on a fresh Chromebrew without those "extra" dependencies?  The only real dependency should be xauth and nothing else.

Reply:
If I installed it without those dependencies, `startx` would not even start.
```
chronos@localhost ~/Download/source/chromebrew/packages $ startx
/usr/local/bin/startx: 186: /usr/local/bin/startx: xauth: not found
/usr/local/bin/startx: 203: /usr/local/bin/startx: xauth: not found
/usr/local/bin/startx: 203: /usr/local/bin/startx: xauth: not found

_XSERVTransmkdir: Owner of /tmp/.X11-unix should be set to root
xinit: XFree86_VT property unexpectedly has 0 items instead of 1
/usr/local/etc/X11/xinit/xinitrc: 52: /usr/local/etc/X11/xinit/xinitrc: xclock: not found
/usr/local/etc/X11/xinit/xinitrc: 51: /usr/local/etc/X11/xinit/xinitrc: twm: not found
```
Without `xterm`:
```
chronos@localhost ~/Download/source/chromebrew/packages $ startx
/usr/local/bin/startx: 186: /usr/local/bin/startx: xauth: not found
/usr/local/bin/startx: 203: /usr/local/bin/startx: xauth: not found
/usr/local/bin/startx: 203: /usr/local/bin/startx: xauth: not found

_XSERVTransmkdir: Owner of /tmp/.X11-unix should be set to root
xinit: XFree86_VT property unexpectedly has 0 items instead of 1
/usr/local/etc/X11/xinit/xinitrc: 55: exec: xterm: not found
xinit: connection to X server lost
```